github/event_types.go

```diff
@@ -106,3 +106,15 @@ type IssueCommentEvent struct {
 	Repo   *Repository `json:"repository,omitempty"`
 	Sender *User       `json:"sender,omitempty"`
 }
+
+// CommitCommentEvent represents the payload delivered by CommitComment webhook.
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#commitcommentevent
+type CommitCommentEvent struct {
+	Comment *RepositoryComment `json:"comment,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Action *string     `json:"action,omitempty"`
```

Comment:
Although it is not specifically listed as being sent in the Event API:
https://developer.github.com/v3/activity/events/types/#events-api-payload
so I moved it to the Webhooks section.  I'm not sure where you see it being not specific to webhooks.